Seminar2/SecondSeminar/src/main/java/sopt/org/secondSeminar/controller/post/PostController.java

```diff
@@ -0,0 +1,76 @@
+package sopt.org.secondSeminar.controller.post;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import sopt.org.secondSeminar.controller.post.dto.request.ReqRegisterPost;
+import sopt.org.secondSeminar.controller.post.dto.response.ResGetPost;
+import sopt.org.secondSeminar.controller.post.dto.response.ResRegisterPost;
+import sopt.org.secondSeminar.domain.Post;
+import sopt.org.secondSeminar.service.PostService;
+
+import java.util.ArrayList;
+
+import static sopt.org.secondSeminar.SecondSeminarApplication.postList;
+
+@RestController
+@RequestMapping("/post")
+@AllArgsConstructor
+public class PostController {
+    private final PostService postService;
+
+    // 포스트 생성
+    @PostMapping("")
+    @ResponseBody
```

Comment:
```suggestion
    // 포스트 생성
    @PostMapping()
    @ResponseBody
```
`""` 리터럴 없애도 되는 것으로 알고있습니다!